### PR TITLE
PR #176 follow up

### PR DESCRIPTION
**Pull request by antocuni**
This is a small followup of #176 to fix a couple of things which were noticed after the PR was already merged.

### CODE REVIEW

File: hpy/debug/src/debug_handles.c [hunk 1]
```diff
@@ -74,23 +74,10 @@ void DHPy_invalid_handle(HPyContext *dctx, DHPy dh)
...
-    uh_res = HPy_CallTupleDict(uctx, info->uh_on_invalid_handle, uh_args, HPy_NULL);
-    if (HPy_IsNull(uh_res)) {
+    uh_res = HPy_CallTupleDict(uctx, info->uh_on_invalid_handle, HPy_NULL, HPy_NULL);
+    if (HPy_IsNull(uh_res))
         print_error(uctx, "Error when executing the on_invalid_handle callback");
-        goto exit;
-    }
- exit:
-    HPy_Close(uctx, uh_args);
-    HPy_Close(uctx, uh_res);
```
Comment (hodgestar) [inline hpy/debug/src/debug_handles.c:93]: Should we not still close `uh_res`?

Comment (antocuni) [reply hpy/debug/src/debug_handles.c:93]: ouch, sure! I simplified the code too much. Ironically, this is the kind of errors that the debug mode would catch, but we can't run the debug mode under itself :)

Comment (hodgestar) [reply hpy/debug/src/debug_handles.c:93]: Hehe. Indeed. :)

**Comment by antocuni** [comment]
@hodgestar I addressed your remarks, I'm going to merge this without further review since it's a very simple PR

**Pull request merged**